packages/calling-stateful-client/src/CallSubscriber.ts

```diff
@@ -65,9 +65,7 @@ export class CallSubscriber {
     this._captionsSubscriber = new CaptionsSubscriber(
       this._callIdRef,
       this._context,
-      _isTeamsMeetingCall(this._call)
-        ? this._call.feature(Features.TeamsCaptions)
-        : this._call.feature(Features.AcsCaptions)
+      this._call.feature(Features.TeamsCaptions)
```

Comment:
Note: don't create captionSubscribre here when it is ACS call

Reply:
fixed